src/socket-gas-helpers/utils.ts

```diff
@@ -0,0 +1,67 @@
+import { Contract, utils } from "ethers";
+import { defaultAbiCoder } from "ethers/lib/utils";
+import { StaticJsonRpcProvider } from "@ethersproject/providers";
+
+export type TxData = {
+  from: string;
+  to: string;
+  data: string;
+};
+
+export type Inputs = {
+  amount: string;
+  receiver: string;
+  executionData: string;
+  connectorPlug: string;
+  dstSocketAddress: string;
+};
+
+export type ChainDetails = {
+  srcChainSlug: number;
+  dstChainSlug: number;
+  dstRPC: string;
+  isArbStackChain: boolean;
+};
```

Comment:
do we need isArbStackChain as input? cant we figure it out from dl core? also, it is unclear if src or dst is arb stack.